README.md

```diff
@@ -27,31 +29,109 @@ npm install @dojo/widgets
 ```
 This package contains *all* of the widgets in this repo.
 
-All of the widgets in this repo are on the same release schedule, that is to say, that we release all widgets at the same time. Minor releases may include new widgets and/or features, whereas patch releases may contain fixes to more than 1 widget.
+All of the widgets are on the same release schedule, that is to say, that we release all widgets at the same time.
+Minor releases may include new widgets and/or features, whereas patch releases may contain fixes to more than 1 widget.
 
-In a future release, we will also publish each widget individually to `npm`.
+To use a widget in your application, you will need to import each widget individually:
+
+```ts
+import Button from '@dojo/widgets/button/Button';
+```
+
+Each widget module has a default export of the widget itself, as well as named exports for things such as properties specific to the widget:
+
+```ts
+import Button, { ButtonProperties } from '@dojo/widgets/button/Button';
+```
+
+Because each widget is a separate module, when you create a release build of your application, you will only include the widgets that you have explicitly imported. 
```

Comment:
Maybe we should state the benefit more explicitly, e.g. "you will only include the widgets that you have explicitly imported, reducing the size of your production application." ?